### ISSUE TITLE
Small hydros/brakes wishlist

### ISSUE THREAD

**Issue by DODGEcharger** [RigsOfRods/rigs-of-rods#2320]
Ulteq myself and others were discussing some improvements to steering (hydros) and brakes, Ulteq suggested I compile the ideas and put them here for discussion/review.

First a steering ratio option for hydros to simulate quick/slow ratio steering boxes.

```
hydros
;node1, node2, extension factor, option, ratio
24, 25, 0.2, n, 3.0
```
3.0 implies a 3:1 ratio relative to the default hydro speed. Alternatively the desired result when using a gamepad/wheel or any axis for control would be a 3x multiplication of the input distance 10% becomes 30% travel, 30 degrees on a wheel becomes 90 in-game, etc. I'm not sure how this split functionality could be achieved but I'm sure there's a solution. For backwards compatibility the default ratio would be left at 1.

Next up is adjustable brake bias and a momentary control for parking the parking brake to allow handbrake use as seen with rally, drifting and other forms of driving.

Brake bias syntax could look something like this:
```
brakes
;brakeforce, parking brake force, wheel 0 bias, wheel 1 bias, wheel 2 bias, wheel 3 bias.....
1500, 800, 0.32, 0.32, 0.68, 0.68
```
The final braking force is just brakeforce*bias. Handbrakes shouldn't require any new truck file syntax, just a new control definition and matching entry in the input map, it should be able to be controlled by both buttons and axis. The default bias would be 1.0 for backwards compatibility.

**Comment by cryham** [comment]
Yeah exactly, I'd also like handbrake, it's quite used in WRC and would be good for all those cars that don't over steer easily.
I could look into adding it. I'd prefer a separate key, like Space for it. IDK P feels for me like more of a truck thing.
